hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -567,6 +574,84 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 		log.Info("Reconciled userData Secret", "result", result)
 	}
 
+	// Validate tunedconfig input.
+	tunedConfig, err := r.getTunedConfig(ctx, nodePool, hcluster)
```

Comment:
hcluster in header is unused.

Reply:
can you please move all this logic up to where we do all the validation and conditions?

Reply:
Thanks, I removed the unused hcluster and moved this logic up next to the other validation and status conditions.